packages/database/drizzle/0032_premium_sprite.sql

```
@@ -0,0 +1,6 @@
+ALTER TABLE "macros" ADD COLUMN "forked_from" uuid;--> statement-breakpoint
+ALTER TABLE "protocols" ADD COLUMN "forked_from" uuid;--> statement-breakpoint
+ALTER TABLE "workbooks" ADD COLUMN "forked_from" uuid;--> statement-breakpoint
+ALTER TABLE "macros" ADD CONSTRAINT "macros_forked_from_macros_id_fk" FOREIGN KEY ("forked_from") REFERENCES "public"."macros"("id") ON DELETE set null ON UPDATE no action;--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)